src/workers/sdk.ts

```diff
@@ -42,7 +42,12 @@ const init = async (
   },
   config: ContractsConfig,
   decimalsMap?: Map<string, number>,
-  cachedData?: string
+  sdkConfig?: {
+    cache?: string;
+    batchPairSize?: number;
+    batchBlockSize?: number;
```

Comment:
I think batchPairSize and batchBlockSize are confusing names. If you disagree than at least it would be grammatically more correct to call it pairBatchSize and blockBatchSize.